battleship-client/src/main/java/com/academy/solid/nie/client/communication/SocketServer.java

```diff
@@ -76,4 +85,25 @@ public void sendPlayerMove(final String move) {
         send(allMoves);
         return receiveAllMovesWithoutSending();
```

Comment:
What is happening here? You send moves and then return moves without sending? I don't understand that :( Should sending be responsibility of another class than SocketServer?